client/gutenboarding/index.ts

```diff
@@ -9,6 +9,29 @@ import page from 'page';
 import { hideMasterbar, main, redirectIfNotEnabled } from './controller';
 import { makeLayout, render as clientRender } from 'controller';
 
+// Don't complain about window.wp.data types in our debug function
+declare const window: any;
+
 export default function() {
-	page( '/gutenboarding', redirectIfNotEnabled, hideMasterbar, main, makeLayout, clientRender );
+	page(
+		'/gutenboarding',
+		redirectIfNotEnabled,
+		( context, next ) => {
+			if ( process.env.NODE_ENV !== 'production' ) {
+				if ( typeof window === 'object' ) {
+					if ( ! window.wp ) {
+						window.wp = {};
+					}
+					if ( ! window.wp.data ) {
+						window.wp.data = require( '@wordpress/data' );
```

Comment:
You can build a proxy object instead which would handle all dependencies you would need in the future.

I did something like that for Jest a long time ago:
https://github.com/WordPress/gutenberg/blob/73754642fb005c797a41b23ad7e8cd06943cd12f/test/setup-wp-aliases.js

Reply:
Let's move this convo to https://github.com/Automattic/wp-calypso/pull/36985#issuecomment-545764515 ?